prac_07/programming_language.py

```diff
@@ -0,0 +1,21 @@
+class ProgrammingLanguage:
+    """CP1403 programming language class"""
+
+    def __init__(self, name: str = "", typing: str = "", reflection: bool = False, year: int = "0000"):
+        """initialise a language type
+
+        Reflection: boolean, False = 'No', True = 'Yes'
```

Comment:
This (True = Yes) seems redundant

Reply:
removed

Reply:
Make sure you use correct data. "0000" is not an int.

Reply:
fixed